src/Satooshi/Bundle/CoverallsV1Bundle/Collector/CiEnvVarsCollector.php

```diff
@@ -14,7 +14,7 @@ class CiEnvVarsCollector
     /**
      * Configuration.
      *
-     * @var Satooshi\Bundle\CoverallsV1Bundle\Config\Configuration
+     * @var \Satooshi\Bundle\CoverallsV1Bundle\Config\Configuration
```

Comment:
yeah, I saw that much of your work that I'm aware that I don't need to raise same concern in multiple places across whole PR, once is enough ;)